cypress/support/pages/profile.pageObgect.js

```diff
@@ -0,0 +1,21 @@
+import PageObject from '../PageObject';
+
+class profilePageObject extends PageObject {
```

Comment:
The class name should be in PascalCase. Change 'profilePageObject' to 'ProfilePageObject'.